### PR TITLE
Reorganization of template + style URGENT

### PR DESCRIPTION
**Pull request by balat**

### CODE REVIEW

File: template.distillery/Makefile.db [hunk 1]
```diff
@@ -22,7 +26,7 @@ ELIOMC            := $(ENV_PSQL) $(ELIOMC)
 ELIOMOPT          := $(ENV_PSQL) $(ELIOMOPT)
 ELIOMDEP          := $(ENV_PSQL) $(ELIOMDEP)
 
-# Ocsigenserver uses the DB settings from %%%PROJECT_NAME%%%.conf.in (which
+# Ocsigenserver uses the DB settings from ee.conf.in (which
```
Comment (dannywillems) [inline template.distillery/Makefile.db]: %%%PROJECT_NAME%%%.conf.in

**Comment by dannywillems** [comment]
And there are some warnings from `os_tools`.
```
File "src/os_tools.eliom", line 38, characters 51-57:
Warning 22: The type of this injected value contains a type variable that could be wrongly inferred.
File "src/os_tools.eliom", line 42, characters 23-24:
Warning 22: The type of this injected value contains a type variable that could be wrongly inferred.
File "src/os_tools.eliom", line 38, characters 51-57:
Warning 22: The type of this injected value contains a type variable that could be wrongly inferred.
File "src/os_tools.eliom", line 42, characters 23-24:
Warning 22: The type of this injected value contains a type variable that could be wrongly inferred.
```

**Comment by balat** [comment]
I'm going to remove Os_tools completely very soon (today).

**Comment by balat** [comment]
Ready to merge (and quite urgent, to avoid conflicts everywhere).

**Comment by dannywillems** [comment]
@balat `Os_tools` will be removed in another PR?

**Comment by dannywillems** [comment]
@balat It doesn't compile (missing `Test_style_drawer`) and still some warnings:
```
File "demo_timepicker.eliom", line 5, characters 2-25:
Warning 33: unused open Eliom_content.Html.
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server -g test_style_config.eliom
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server -g demo_rpc.eliom
File "demo_carousel3.eliom", line 63, characters 21-30:
Warning 27: unused variable swipe_pos.
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server -g demo_calendar.eliom
js_of_eliom -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.client -g demo_popup.eliom
File "demo_carousel1.eliom", line 7, characters 2-25:
Warning 33: unused open Eliom_content.Html.
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -infer -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server test_style_services.eliom
js_of_eliom -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.client -g demo_timepicker.eliom
File "demo_rpc.eliom", line 7, characters 2-25:
Warning 33: unused open Eliom_content.Html.
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -infer -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server test_style_base.eliom
File "demo_calendar.eliom", line 7, characters 2-25:
Warning 33: unused open Eliom_content.Html.
js_of_eliom -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.client -g test_style_icons.eliom
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server -g test_style_icons.eliom
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server -g test_style_base.eliom
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -infer -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server test_style_mobile.eliom
eliomc -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -infer -package lwt.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.server test_style_container.eliom
js_of_eliom -w +A-4-7-9-37-38-39-41-42-44-45-48 -ppx -c -package lwt.ppx -package js_of_ocaml.ppx -package js_of_ocaml.deriving.ppx -package ppx_deriving.std -package ocsigen-start.client -g demo_carousel3.eliom
File "test_style_container.eliom", line 94, characters 11-33:
Error: Unbound module Test_style_drawer
Makefile.os:215: recipe for target '_server/test_style_container.type_mli' failed
make: *** [_server/test_style_container.type_mli] Error 2
make: *** Waiting for unfinished jobs....
File "demo_carousel3.eliom", line 63, characters 21-30:
Warning 27: unused variable swipe_pos.
```

**Comment by vasilisp** [comment]
I glimpsed through the code and didn't spot anything wrong. Merge it when you feel it's ready. We can fix the warnings later.

**Pull request closed without merging**